arangod/Replication2/Storage/RocksDB/LogPersistor.h

```diff
@@ -68,6 +73,7 @@ struct LogPersistor final : ILogPersistor {
   std::shared_ptr<AsyncLogWriteBatcherMetrics> const _metrics;
   ::rocksdb::DB* const db;
   ::rocksdb::ColumnFamilyHandle* const logCf;
+  arangodb::ICompactKeyRange* _engine;
```

Comment:
I would prefer a different name since we don't really have the engine - after all you did the right thing and applied the [interface segregation principle](https://en.wikipedia.org/wiki/Interface_segregation_principle), so we have an `ICompactKeyRange` instance that can only do `compactRange`. So can we call it "compactor" or something like that?

Reply:
Absoutely, will rename